src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksConstants.java

```diff
@@ -84,6 +84,12 @@
         public static final String DESCRIPTION_SUFFIX_FOR_NEW_VENDOR = "--new vendor";
         public static final Integer DESCRIPTION_MAX_LENGTH = 40;
         public static final String PAYMENTWORKS_NEW_VENDOR_CREATE_ROUTE_ANNOTATION = "Generated by PaymentWorks New Vendor Create Batch Process";
+        
+        public static final class KFSPoVendorConstants {
+            public static final String PAYMENT_TERMS_CODE = "00N30";
+            public static final String VENDOR_SHIPPING_TITLE_CODE = "DE";
+            public static final String VENDOR_SHIPPING_PAYMENT_TERMS_CODE = "PA";
```

Comment:
I think these variables should be more descriptive, such as payment_term_code_net_30_day